alerter/alerter.cs

```diff
@@ -24,6 +25,7 @@ class Alerter {
         static void Main(string[] args) {
             alertInCelcius(400.5f);
             alertInCelcius(303.6f);
+            Debug.Assert(alertFailureCount > 0);

```

Comment:
This is a weak test check. What if the expected value is 2 and the production code returns 1?
Such bugs cannot be caught with this check.
The test should know exactly what is the output expected from the production code.